### PR TITLE
(core) Upgrade to Richie 1.9.2

### PR DESCRIPTION
**Pull request by deodat**
1.9.2 - 2019-09-20

Fixed

Logo image location wrongly targetted in base.html new SEO tags.

1.9.1 - 2019-09-20

Added

Add an UPGRADE.md file to document upgrade instructions.

Changed

Update and include the latest version of translations.

1.9.0 - 2019-09-18

Added

Add a React-powered course search field that can live anywhere in Richie.
Add breadcrumbs element on all pages but the listing ones.
Add a new page type for programs as a collection of courses.
New dependency "dj-pagination" to enable pagination on blogpost list and person list.
Add basic SEO tags with some more relevant ones in page details.

Changed

Refactor course search field to make it reusable.
Upgrade crowdin image used in circle-ci to version 2.0.31 including tar command.

Fixed

Translate login/signup on header.

### CODE REVIEW

File: src/frontend/package.json [hunk 1]
```diff
@@ -32,6 +32,6 @@
     "node-sass": "4.11.0",
     "nodemon": "1.18.10",
     "prettier": "1.16.4",
-    "richie-education": "^1.8.3"
+    "richie-education": "^1.9.2"
```
Comment (mbenadda) [inline src/frontend/package.json]:
```suggestion
    "richie-education": "1.9.2"
```

And you'll need to re-run `yarn install` to update the lock file to this change :)

Comment (deodat) [reply src/frontend/package.json]: Ok done !

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (jmaupetit) [inline yarn.lock]: I think this file shouldn't exists nor have been committed, right?

Comment (deodat) [reply yarn.lock]: yes, indeed, I made `yarn install `in the wrong directory :(
resolved

**Pull request merged**